oauth2client/file.py

```diff
@@ -20,13 +20,20 @@
 
 import os
 import threading
+import warnings
 
 from oauth2client import client
 
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
 
+_IS_DIR_MESSAGE = '{0}: Is a directory'
+
```

Comment:
I was going by the blank lines guidence in [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines)
> Surround top-level function and class definitions with two blank lines.

But after seeing it explicitly says functions and classes, this may not need to follow that.